felix/jitter/jitter_suite_test.go

```diff
@@ -31,6 +29,5 @@ func init() {
 
 func TestJitter(t *testing.T) {
 	RegisterFailHandler(Fail)
-	junitReporter := reporters.NewJUnitReporter("../report/jitter_suite.xml")
```

Comment:
Are we losing our junit reports with this change?

Reply:
no, just done on command line rather then in the code. This code is gone.

Reply:
Oh right, I see the corresponding change now.